pkg/util/github/repo.go

```diff
@@ -17,3 +21,34 @@ func (c *Client) CreateRepo() error {
 	}
 	return nil
 }
+
+func (c *Client) DeleteRepo() error {
+	_, err := c.Client.Repositories.Delete(
```

Comment:
just `_, err := c.Client.Repositories.Delete(c.Context, c.Owner, c.Repo)` is fine. same for below